### PR TITLE
Mention

### PR DESCRIPTION
**Pull request by SharonStrats**
Steps:
1) Trigger the mention -- done
2) Create a context for showing the people that can be mentioned.  -- done
3) Attach the mention to the message -- see sentiment functionality -- todo
4) Notify mentionee --todo

### CODE REVIEW

File: src/chat/infinite.js [hunk 1]
```diff
@@ -286,6 +346,26 @@ export function infiniteMessageArea (dom, kb, chatChannel, options) {
         'keydown',
         async function (e) {
           // User preference?
+          debug.log('key code ' + e.keyCode)
+          if (e.shiftKey) {
+            if (e.keyCode === 50) {
+              debug.log('testing @ key')
```
Comment (angelo-v) [inline src/chat/infinite.js:352]: You are checking for Shift+2, wich is a `"` on a german keyboard, not an `@`. I think you need to check if the @ character is typed, regardless of the keycode that produced it.

File: src/chat/infinite.js [hunk 2]
```diff
@@ -155,6 +206,15 @@ export function infiniteMessageArea (dom, kb, chatChannel, options) {
     form.appendChild(rhs)
     form.AJAR_date = '9999-01-01T00:00:00Z' // ISO format for field sort
     var field, sendButton
+    debug.log('line 158')
+    var parps = kb
+      .each(chatChannel, ns.wf('participation'))
+      .map(function (parp) {
+        const name = parp.value.replace('/https:///', '').split('.')
+        debug.log('parp ' + name[0])
```
Comment (angelo-v) [inline src/chat/infinite.js:214]: Why not use the real name of the WebID?

File: src/chat/infinite.js [hunk 3]
```diff
@@ -88,6 +88,57 @@ function desktopNotification (str) {
   // want to be respectful there is no need to bother them any more.
 }
 
+// HELPER CODE FOR MENTION - WILL NEED TO MOVE
```
Comment (angelo-v) [inline src/chat/infinite.js:91]: In general I am missing automated tests. Shall I help with that?

**Comment by SharonStrats** [comment]
Closing this for now, but leaving the branch. Need to convert to div before it's possible.

**Pull request closed without merging**